Rover/sailboat.cpp

```diff
@@ -102,6 +102,13 @@ const AP_Param::GroupInfo Sailboat::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("LOIT_RADIUS", 9, Sailboat, loit_radius, 5),
 
+    // @Param: WNDVN_ROTATES
+    // @DisplayName: Windvane on rotating mast
+    // @Description: Leave at 0 if windvane is fixed to hull; set 1 if vane fixed to rotating mast
+    // @Values: 0:Fixed to hull,1:Rotates with mast
+    // @User: Standard
+    AP_GROUPINFO("WNDVN_ROTATES", 10, Sailboat, wndvn_rotates, 0),
```

Comment:
Your only allowed 16 chars total for a param name, these automatically get the `SAIL_` prefix added so your 2 too long

Reply:
I think it should also be in the wind-vane library, other wise logging and MAVLink will all be wrong. I don't think its going to work that well unless we add sport for precise angle measurement of the mast.

Reply:
Maybe I leave it alone for now then and rig a mechanical solution to my use case.

Reply:
Feel free to give it a try, the disadvantage is that the wind reading accuracy would be the accuracy of the mast angle + the accuracy of the wing sensor. Of course if your mast angle sensor/estimate is good than that is not a problem. That is certainly something I would like to support in the future.

Reply:
My assumption was that knowing the servo command is a close enough approximation for the wind angle--perhaps more so for a direct-drive mast than a winch. Of course the way you set the winch is by deriving the angle you want the boom to rotate to. But i should probably get _something_ working before I pile on features. For future reference, how can I find what was the last command we sent to a servo?

Reply:
`SRV_Channels::get_output_scaled(SRV_Channel::k_mast_rotation)`. But because the mounted on the mast flag should live in windvane and the mast angle stuff is a rover value it might be easier to just pass the offset to AP_WIndvane in a new function.

Reply:
That was my thinking behind putting it under `SAIL`: winch and mast_rotation branches of `get_throttle_and_mainsail_out` would check the parameter and it it's true, apply the correction right there without writing the computed value anywhere.